broker/setup_broker/lvk/setup_broker.sh

```diff
@@ -71,13 +75,14 @@ if [ "${teardown}" != "True" ]; then
     # create pubsub
     echo "Configuring Pub/Sub resources..."
     gcloud pubsub topics create "${topic_alerts}"
-    gcloud pubsub topics create "${topic_storebigquery}"
+    gcloud pubsub topics create "${topic_deadletter}"
+    gcloud pubsub subscriptions create "${subscription_storebigquery}" --topic="${topic_alerts}" --bigquery-table="${PROJECT_ID}:${bq_dataset}.${alerts_table}" --use-table-schema --drop-unknown-fields --dead-letter-topic="${topic_deadletter}" --max-delivery-attempts=5 --dead-letter-topic-project="${PROJECT_ID}"
```

Comment:
```suggestion
    gcloud pubsub subscriptions create "${subscription_storebigquery}" \
        --topic="${topic_alerts}" \
        --bigquery-table="${PROJECT_ID}:${bq_dataset}.${alerts_table}" \
        --use-table-schema \
        --drop-unknown-fields \
        --dead-letter-topic="${topic_deadletter}" \
        --max-delivery-attempts=5 \
        --dead-letter-topic-project="${PROJECT_ID}"
```

Try to keep lines under about 100 characters (or whatever number we have black set to for the python files).